ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/valueobjects/LdesFragmentIdentifier.java

```diff
@@ -69,6 +70,27 @@ public String getFragmentId() {
 		return stringBuilder.toString();
 	}
 
+	public String getParentId() {
+
```

Comment:
Since we are now working with FragmentIdentifiers, maybe this method should return an LdesFragmentIdentifier instead of a String? a part of the string concatenation logic below would then be de-duplicated as you would then call .asString on the parent identifier object.